homeassistant/requirements.py

```diff
@@ -47,6 +47,8 @@ def pip_kwargs(config_dir: Optional[str]) -> Dict[str, Any]:
     kwargs = {
         'constraints': os.path.join(os.path.dirname(__file__), CONSTRAINT_FILE)
     }
-    if not (config_dir is None or pkg_util.is_virtual_env()):
+    if 'HASSIO_WHEELS' in os.environ:
+        kwargs['find_links'] = os.environ['HASSIO_WHEELS']
+    elif not (config_dir is None or pkg_util.is_virtual_env()):
```

Comment:
I don't agree with this change to `elif`. If we don't want to use `deps` under Docker, we should check explicitly for being in Docker using `os.path.isfile('/.dockerenv')` (maybe put it in a `util/docker.py`).

Reply:
I don't export it to `util/docker.py` because I do it like `is_virtual_env`